### ISSUE TITLE
[REQ] Souvenir Cases

### ISSUE THREAD

**Issue by SkyJust4Fun** [ianlucas/cs2-inventory-simulator#94]
I got told that souvenir packages don't drop stattrak item types :3
Not even that urgent or that crazy to be fixed/enhanced, but for the nerds that go crazy about it.

**Comment by ianlucas** [comment]
We had an open issue related to rework on Souvenir Packages, but I think the user that initially opened it has closed their account on GitHub. I'm going to use this issue to keep track of it.

**Comment by KewaiiGamer** [comment]
I think souvenir cases are completely lacking the stickers that are on souvenir skins. though this migth be more suitable for cs2-lib and not cs2-inventory-simlator

**Comment by ianlucas** [comment]
> I think souvenir cases are completely lacking the stickers that are on souvenir skins. though this migth be more suitable for cs2-lib and not cs2-inventory-simlator

Yeah the issue is open here, but usually the logic will live on cs2-lib.
